Document the MCP editor server for starting Promptless tasks (#861)
Promptless/docs: src/content/docs/docs/connect/triggers/api.mdx

API triggers let external systems request documentation updates from Promptless. Connect your CI/CD pipelines, custom automation tools, or any system that can make HTTP requests.

<Aside type="tip">

>   For interactive editor use without managing an API key, connect over MCP instead. Choose by caller: an API key suits server-to-server and CI callers, while [MCP triggers](/docs/connect/triggers/mcp) suit your editor with no key to create or paste.

Reviewer: ```suggestion
  For interactive editor use without managing an API key, connect over MCP instead. Use API keys for server-to-server and CI callers, and use [MCP triggers](/docs/connect/triggers/mcp) for IDEs and agents.
```
